diff --git a/Kui_Nameplates/create.lua b/Kui_Nameplates/create.lua
index 846949b..39fbfa1 100644
--- a/Kui_Nameplates/create.lua
+++ b/Kui_Nameplates/create.lua
@@ -54,6 +54,12 @@ function addon:CreateHealthBar(frame, f)
     f.health:SetStatusBarTexture(addon.bartexture)
 	f.health.percent = 100
 
+    f.health.bg = CreateFrame('StatusBar',nil,f)
+    f.health.bg:SetStatusBarTexture(addon.bartexture)
+    f.health.bg:SetAllPoints(f.health)
+
+    f.health:SetFrameLevel(f.health.bg:GetFrameLevel()+1)
+
     if self.SetValueSmooth then
         -- smooth bar
         f.health.OrigSetValue = f.health.SetValue
diff --git a/Kui_Nameplates/layout.lua b/Kui_Nameplates/layout.lua
index 78a68f2..d933b9d 100644
--- a/Kui_Nameplates/layout.lua
+++ b/Kui_Nameplates/layout.lua
@@ -105,7 +105,8 @@ local function SetHealthColour(self,sticky,r,g,b)
             self.player = true
         end
 
-        self.health:SetStatusBarColor(r, g, b)
+        self.health.bg:SetStatusBarColor(r,g,b)
+        self.health:SetStatusBarColor(0,0,0,.8)
     end
 end
 
